ui/app/src/modules/Content/History/HistoryDialog.tsx

```diff
@@ -392,6 +405,7 @@ function HistoryDialog(props: HistoryDialogProps) {
         <SuspenseWithEmptyState resource={versionsResource}>
           <VersionList
             resource={versionsResource}
+            permissionsResource={permissionsResource}
```

Comment:
change resource prop to be `{ versions: Resource<...>; permissions: Resource<...>; } `